test/inline_asm_clobbers.cl

```diff
@@ -0,0 +1,86 @@
+// RUN: %clang_cc1 -triple spir64-unknown-unknown -x cl -cl-std=CL2.0 -O0 -fdeclare-opencl-builtins -finclude-default-header -emit-llvm-bc %s -o %t.bc
```

Comment:
`-finclude-default-header` slowdowns the test: since the only built-in you are using is `get_global_id`, I suggest to remove all built-in-related options from the command and just add forward-declaration for `get_global_id`

Reply:
I will inevitably need headers.

`size_t __ovld __cnfn get_global_id(uint dimindx);`

uses size_t which is platform dependent.

Reply:
Done